trikot-viewmodels-declarative/compose/build.gradle.kts

```diff
@@ -50,25 +51,26 @@ dependencies {
 }
 
 tasks {
-    val sourcesJar by creating(Jar::class) {
+    val sourcesJar by registering(Jar::class) {
+        val sourceSet = android.sourceSets.getByName("main").kotlin as DefaultAndroidSourceDirectorySet
+        from(sourceSet.srcDirs)
         archiveClassifier.set("sources")
-        from(android.sourceSets.getByName("main").java.srcDirs().toString())
+    }
+
+    artifacts {
+        archives(sourcesJar)
     }
 }
 
+
```

Comment:
Is this wanted? There doesn’t seem to be other double blank lines anywhere else in the file.